components/event-bus/apis/applicationconnector/v1alpha1/register.go

```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
-	remoteenvironmentkymaio "github.com/kyma-project/kyma/components/event-bus/internal/ea/apis/applicationconnector.kyma-project.io"
+	remoteenvironmentkymaio "github.com/kyma-project/kyma/components/event-bus/apis/applicationconnector"
```

Comment:
That alias doesn't match the package name (my fault, sorry)

Reply:
will just remove it